### PR TITLE
increase sequence and add "?" option

### PR DESCRIPTION
**Pull request by ibrahimshibani**
- Increase Fibonacci sequence size. 
- Add a "?" as a  "I don't know" option.

### CODE REVIEW

File: src/views/planningPoker.tsx [hunk 1]
```diff
@@ -5,24 +5,32 @@ import { PlanningPokerStyles } from './planningPokerStyles';
 const EXTENSION_ID = 'aha-develop.planning-poker';
 const FIELD_BASE = 'estimate';
 const ESTIMATES = {
-  '0': { color: '#666666', backgroundColor: '#f1f1f1' },
+  '?': { color: '#666666', backgroundColor: '#f1f1f1' },
```
Comment (percyhanna) [inline src/views/planningPoker.tsx]: Some of our customers may choose to continue scoring stories as 0 points. Could we keep the `0` option, and add `?` as a new option instead?

File: src/views/planningPoker.tsx [hunk 2]
```diff
@@ -5,24 +5,32 @@ import { PlanningPokerStyles } from './planningPokerStyles';
...
   '2': { color: '#301c42', backgroundColor: '#e5dced' },
   '3': { color: '#7d630b', backgroundColor: '#faebb9' },
   '5': { color: '#c76d00', backgroundColor: '#fcddb8' },
   '8': { color: '#992e0b', backgroundColor: '#fac0af' },
+  '13': { color: '#992e0b', backgroundColor: '#ea5427' },
+  '21': { color: '#ffffff', backgroundColor: '#dc143c' },
+  '34': { color: '#f5f5f5', backgroundColor: '#990000' },
+  '55': { color: '#eeeeee', backgroundColor: '#68228b' },
+  '89': { color: '#ffffff', backgroundColor: '#151515' },
+
```
Comment (percyhanna) [inline src/views/planningPoker.tsx]: For the time being, we would like to keep the voting options to smaller numbers. In the future, we may consider making these points configurable as part of the extension settings.

```suggestion
```

File: src/views/planningPoker.tsx [hunk 3]
```diff
@@ -93,20 +101,40 @@ const VoteList = ({ votes }) => (
...
+    var avg = "?"
+    var max = "?"
+
+    if (estimates.length !== 0) {
+    
+    min = Math.min.apply(Math, estimates);
+    max = Math.max.apply(Math, estimates);
+    var sum = 0;
+    estimates.forEach(function(num) { sum += num });
+    avg = sum / estimates.length;
```
Comment (percyhanna) [inline src/views/planningPoker.tsx]: Could we keep our original calculation methods here instead?

```suggestion
    min = Math.min(...estimates);
    avg = estimates.reduce((n, sum) => n + sum, 0) / estimates.length;
    max = Math.max(...estimates);
```

File: src/views/planningPoker.tsx [hunk 4]
```diff
@@ -93,20 +101,40 @@ const VoteList = ({ votes }) => (
...
 
 const VoteAnalysis = ({ votes }) => {
-  const estimates = votes.map((v) => v.estimate);
-  const min = Math.min(...estimates);
-  const avg = estimates.reduce((n, sum) => n + sum, 0) / estimates.length;
-  const max = Math.max(...estimates);
 
-  return (
+  const estimates = []
+//  const estimates = votes.map((v) => v.estimate);
```
Comment (percyhanna) [inline src/views/planningPoker.tsx]: Could you remove this comment, please?

```suggestion
```

**Review by percyhanna — CHANGES_REQUESTED** [review]
Hi Ibrahim,

I'm Percy, on the engineering team here at Aha!

Thanks so much for opening up this PR — we really appreciate your contributions and helping us improve our open-source extensions.

For the time being, we would like to be mindful that many teams might use this extension out of the box, so we would like to keep some of the larger story points outside of the default extension. In the future, we may consider adding more settings to customize the points available for estimating.

Thanks again for the suggestions!

File: src/views/planningPoker.tsx (hunk 2 above)
Comment (ibrahimshibani) [reply src/views/planningPoker.tsx]: kept the original range

File: src/views/planningPoker.tsx (hunk 3 above)
Comment (ibrahimshibani) [reply src/views/planningPoker.tsx]: done

**Review by percyhanna — APPROVED** [review]
Thanks for the PR and the changes 👍

**Pull request merged**